src/Features/CSharp/Portable/SemanticSearch/CSharpSemanticSearchService.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if NET6_0_OR_GREATER
```

Comment:
Should remove this conditional

Reply:
Filed https://github.com/dotnet/roslyn/issues/72617 as a follow up.